frontend/tests/ui/integration-test.py

```diff
@@ -57,9 +57,9 @@ def create_user(self):
     assert username_label.text == self.username
 
   def create_problem(self):
-    nav_problems = self.driver.find_element_by_css_selector('#nav-problems')
+    nav_problems = self.driver.find_element_by_css_selector('.nav-problems')
```

Comment:
```suggestion
    nav_problems = self.driver.find_element_by_css_selector('#common-navbar .nav-problems')
```

esto funciona? hay manera de poner algún ID antes de la clase? igual en todos los CSSs selectors / xpath expressions de este cambio.

Reply:
Como tal, no se puede utilizar `#common-navbar`, pero podemos utilizar `#root`